src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -53,7 +54,7 @@ public static AuthenticationBuilder AddScriptJwtBearer(this AuthenticationBuilde
 
                             o.TokenValidationParameters = CreateTokenValidationParameters();
 
-                            if (!WebScriptHostManager.InStandbyMode)
+                            if (!new ScriptWebHostEnvironment().InStandbyMode)
```

Comment:
Mark this with a TODO as well

Reply:
This should be covered by the TODO above (as the model here will change), but will add a comment.